auth-api/src/auth_api/resources/org.py

```diff
@@ -158,7 +158,8 @@ def put(org_id):
                     response, status = org.change_org_ype(request_json, action,
                                                           bearer_token).as_dict(), http_status.HTTP_200_OK
                 else:
-                    response, status = org.update_org(request_json, bearer_token).as_dict(), http_status.HTTP_200_OK
+                    response, status = org.update_org(request_json, token, bearer_token).as_dict(),\
```

Comment:
trivial : May be rename token to user_info or something? token and bearer token are confusing, and I thought both are same.

Reply:
yeah.makes sense

Reply:
i changed it.